packages/trader/src/Modules/MT5/Components/mt5-demo-account-display.jsx

```diff
@@ -37,119 +37,111 @@ const MT5DemoAccountDisplay = ({
         </div>
     ) : (
         <div className='mt5-demo-accounts-display'>
-            {landing_companies?.mt_gaming_company?.financial && (
-                <MT5AccountCard
-                    has_mt5_account={has_mt5_account}
-                    icon={() => <Icon icon='IcMt5SyntheticPlatform' size={64} />}
-                    title={localize('Synthetic')}
-                    type={{
+            <MT5AccountCard
```

Comment:
Was this decided to show all cards for `eu` as well?

Reply:
There's a bit of lack of clarification for this but I believe `eu` was already handled in #1446 as per our group discussion.